test/requirements-test.txt

```diff
@@ -16,11 +16,7 @@ pytorch_lightning
 tabulate
 torchvision
 datasets
-azureml-evaluate-mlflow
-mlflow
-# newest version mlflow-skinny==2.4.0 is not compatible with the test
-# issue: https://github.com/mlflow/mlflow/issues/8629
-mlflow-skinny==2.3.2
+azureml-evaluate-mlflow>=0.0.14
```

Comment:
azureml-evaluate-mlflow will force the version of mlflow-skinny to 2.3.1. So I removed it.